File: datasets/IR-Plag/convertai-original-typescript-rest-java/case-06/plagiarized/L4/05/singlearray.java
package IR;

import java.util.*;

class singlearray
{
	static int[] n = new int[10];
	static void print()
	{
				for (int a = 9; a >= 0; a--) 
		{
			System.out.println(n[a]);
		}
	}
	public static void main (String[] args)
	{
				
		Scanner inp = new Scanner(System.in);

		for (int i = 0; i < 10; i++) 
		{
			System.out.print("Read a number: ");
			n[i] = inp.nextInt();
		}
print();
	}
}

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-06/plagiarized/L4/05/singlearray.kt
package IR

import java.util.Scanner

object singlearray {
    private val n = IntArray(10)

    @JvmStatic
    fun print() {
        for (a in 9 downTo 0) {
            println(n[a])
        }
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val inp = Scanner(System.`in`)

        for (i in 0 until 10) {
            print("Read a number: ")
            n[i] = inp.nextInt()
        }

        print()
    }
}

